src/pages/docs/octopus-rest-api/octopus.server.exe-command-line/path.md

```diff
@@ -62,4 +68,9 @@ octopus.server path --taskLogs \\Octoshared\OctopusData\TaskLogs
 octopus.server path --nugetRepository \\Octoshared\OctopusData\Packages
 octopus.server path --imports \\Octoshared\OctopusData\Imports
 octopus.server path --telemetry \\Octoshared\OctopusData\Telemetry
+octopus.server path --eventExports \\Octoshared\OctopusData\EventExports
 ```
+
+:::div{.hint}
```

Comment:
I don't think this hint block is needed here

Reply:
Deleted